db/blip_sync_context.go

```diff
@@ -191,11 +174,9 @@ func (bsc *BlipSyncContext) register(profile string, handlerFn func(*blipHandler
 				response.SetError("HTTP", status, msg)
 			}
 			base.InfofCtx(bsc.loggingCtx, base.KeySyncMsg, "#%d: Type:%s   --> %d %s Time:%v", handler.serialNumber, profile, status, msg, time.Since(startTime))
-		} else {
 			// Log the fact that the handler has finished, except for the "subChanges" special case which does it's own termination related logging
-			if profile != "subChanges" {
-				base.DebugfCtx(bsc.loggingCtx, base.KeySyncMsg, "#%d: Type:%s   --> OK Time:%v", handler.serialNumber, profile, time.Since(startTime))
-			}
+		} else if profile != "subChanges" {
+			base.DebugfCtx(bsc.loggingCtx, base.KeySyncMsg, "#%d: Type:%s   --> OK Time:%v", handler.serialNumber, profile, time.Since(startTime))
```

Comment:
move comment down from above

```suggestion
			// Log the fact that the handler has finished, except for the "subChanges" special case which does it's own termination related logging
			base.DebugfCtx(bsc.loggingCtx, base.KeySyncMsg, "#%d: Type:%s   --> OK Time:%v", handler.serialNumber, profile, time.Since(startTime))
```